utils/PasswordValidator.js

```diff
@@ -0,0 +1,14 @@
+export function passwordValidator(password) {
+  if (!password) return "Password can't be empty.";
+  if (password.length < 5)
+    return "Password must be at least 5 characters long.";
+  if (password.length > 50)
+    return "Password must be at most 50 characters long.";
+  if (!/[0-9]/.test(password))
+    return "Password must contain at least one digit.";
+  if (!/[a-zA-Z]/.test(password))
+    return "Password must contain at least one letter.";
+  if (/\s/.test(password)) return "Password must not contain white space.";
```

Comment:
How about other special characters? We want to prevent from SQL Injection as soos as possible. Probably it is done well bellow, but we do not want to allow characters like:
",':/  etc.